### PR TITLE
feat(projects): add default set of permissions when creating new project (DSP-1347)

### PR DESCRIPTION
**Pull request by SepidehAlassi**
resolves DSP-1347

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala [hunk 1]
```diff
@@ -613,16 +613,19 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
...
-        checkResult <- administrativePermissionForProjectGroupGetADM(
+        checkResult: Option[AdministrativePermissionADM] <- administrativePermissionForProjectGroupGetADM(
           createRequest.forProject,
           createRequest.forGroup,
           requestingUser = KnoraSystemInstances.Users.SystemUser
         )
 
         _ = checkResult match {
-          case Some(ap) =>
+          case Some(ap: AdministrativePermissionADM) =>
```
Comment (subotic) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala:623]: good catch!

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala [hunk 2]
```diff
@@ -968,6 +971,84 @@ class ProjectsResponderADM(responderData: ResponderData) extends Responder(respo
...
+            )
+          ),
+          featureFactoryConfig = featureFactoryConfig,
+          requestingUser = requestingUser,
+          apiRequestID = UUID.randomUUID()
+        )).mapTo[DefaultObjectAccessPermissionCreateResponseADM]
+
+        // Give the members of the new project rights to modify, view, and restricted view of all resources and values
+        // that belong to the project.
+        doapForProjctAdmin <- (responderManager ? DefaultObjectAccessPermissionCreateRequestADM(
```
Comment (subotic) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala:1035]: Shouldn't this be named `doapForProjectMember`?

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala [hunk 3]
```diff
@@ -968,6 +971,84 @@ class ProjectsResponderADM(responderData: ResponderData) extends Responder(respo
...
+            hasPermissions = Set(PermissionADM.ProjectResourceCreateAllPermission)
+          ),
+          featureFactoryConfig = featureFactoryConfig,
+          requestingUser = requestingUser,
+          apiRequestID = UUID.randomUUID()
+        )).mapTo[AdministrativePermissionCreateResponseADM]
+
+        // Give the admins of the new project rights to change rights, modify, delete, view,
+        // and restricted view of all resources and values that belong to the project.
+        doapForProjctAdmin <- (responderManager ? DefaultObjectAccessPermissionCreateRequestADM(
```
Comment (subotic) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala:1015]: typo: doapForProjectAdmin

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala [hunk 4]
```diff
@@ -968,6 +971,84 @@ class ProjectsResponderADM(responderData: ResponderData) extends Responder(respo
...
+            hasPermissions =
+              Set(PermissionADM.ProjectAdminAllPermission, PermissionADM.ProjectResourceCreateAllPermission)
+          ),
+          featureFactoryConfig = featureFactoryConfig,
+          requestingUser = requestingUser,
+          apiRequestID = UUID.randomUUID()
+        )).mapTo[AdministrativePermissionCreateResponseADM]
+
+        // Give the members of the new project rights to create resources.
+        apPermissionForProjectAdmin: AdministrativePermissionCreateResponseADM <- (responderManager ? AdministrativePermissionCreateRequestADM(
```
Comment (subotic) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala:1001]: apPermissionsForProjectAdmin -> apPermissionsForProjectMember

**Review by subotic — APPROVED** [review]
Thanks! Just a few minor things.

**Comment by SepidehAlassi** [comment]
@subotic Thanks for reviewing this and pointing out the typos! :-D

**Pull request merged**